### DISCUSSION TITLE
Add Ubuntu 24.10 option for os selection

### DISCUSSION THREAD

**Discussion by arteee87** [community-scripts/ProxmoxVE#497] [Ideas]
24.10 is available via pveam.
I've looked over the /misc/build.func and /ct/creacte_lxc.sh and seems that it'll only require to modify 3 lines in /misc/build.func

![изображение](https://github.com/user-attachments/assets/4370471f-6130-4859-b08b-8d06828d4978)

**Comment by MickLesk** [comment]
I'll include it in my next build.func PR. Probably in the next 2-4 days.

**Reply by clipz98** [reply]
Hello, any update on this?

**Reply by MickLesk** [reply]
Look at PRs

**Reply by clipz98** [reply]
> Look at PRs

Thank you. It took me a minute to realize PR = "Pull request" there 😅. Excellent stuff, thanks for the hard work!
For anyone else curious, here's a  link to said PR: https://github.com/community-scripts/ProxmoxVE/pull/643
